packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -242,6 +244,7 @@ - (instancetype)initWithPlayerItem:(AVPlayerItem*)item frameUpdater:(FLTFrameUpd
 
   _player = [AVPlayer playerWithPlayerItem:item];
   _player.actionAtItemEnd = AVPlayerActionAtItemEndNone;
+  [self setInternalPlaybackSpeed:1.0f];
```

Comment:
Why is this needed on init?

Also, per style guide, this should not call self: https://google.github.io/styleguide/objcguide.html#avoid-messaging-the-current-object-within-initializers-and--dealloc
(There are existing violations, but they are wrong, and we shouldn't add more.)

Reply:
Good point, this is not needed and I will remove.